Formula/ghz-web.rb

```diff
@@ -0,0 +1,21 @@
+class GhzWeb < Formula
+  desc "Web interface for ghz"
+  homepage "https://ghz.sh"
+  url "https://github.com/bojand/ghz/archive/v0.52.0.tar.gz"
+  sha256 "6f7e4e0cd81ad00c2c4e17c93257810e2ea8897f60713352c9d9583231e2739a"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "1"
```

Comment:
In case of `ghz-web` I believe we do need CGO enabled because it depends on [go sqlite3 module](https://github.com/mattn/go-sqlite3). Perhaps we would also need xcode in the formula?
```
depends_on :xcode => :build
```